### DISCUSSION TITLE
"Preview In Browser?"

### DISCUSSION THREAD

**Discussion by allstar87** [jendrikseipp/rednotebook#682] [Q&A]
A recent update has changed how a few things about the app work; my word cloud is missing now, and the "preview/Edit" button has been replaced with a button that says "preview in browser" and the app is now permanently in edit mode.

In other words, Rednotebook went from barely adequate to not adequate.  I wasn't really happy with the "edit in almost-markup and then "preview" in rich text" thing, now it can't do rich-text preview within the app at all?  I'm not even sure what features the app even has anymore.  What value does it provide over a plaintext editor at this point?

**Comment by jendrikseipp** [comment]
Thanks for the report! This was just an oversight when I updated the underlying libraries for the flatpak version. Will be fixed in #681.

**Reply by aphelei** [reply]
It still shows "preview in browser" in v2.29.4

**Reply by jendrikseipp** [reply]
Can you please paste the full log when you run it in a terminal?

**Reply by aphelei** [reply]
I'm not on linux I have windows 10. How can I get the logs ? I tried to launch using start 

In v2.29.4 (from github) most button doesn't work. The data are not read either.
In v2.29.3 everythings seems to be working but it has "preview in browser".

In v2.26 everything seems to be working (I like your program so I'll use this version). "Preview in browser" appeared in 2.27

Sorry if it's not very clear.

**Reply by allstar87** [reply]
UPdated to 2.29.4 via Flatpak.  Looks like the issues have been corrected.  Thank you.

**Comment by jendrikseipp** [comment]
I fixed the error (#686) now. Thanks for spotting it!

**Reply by aphelei** [reply]
v2.29.5 still has "preview in browser" (windows)

**Reply by jendrikseipp** [reply]
That's to be expected, unfortunately. There's no suitable GUI browser widget for Windows...

**Reply by Nols78** [reply]
Hello, I also have the same issue.
Just a question out of interest: What was used in v2.27 when it was still working without the "Preview In Browser" option?

**Reply by jendrikseipp** [reply]
There we used CEFPython. Unfortunately, that library doesn't work with any recent GTK version.
